cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -3859,7 +3859,7 @@
 
   <property>
     <name>security.authorization.cache.max.entries</name>
-    <value>100000</value>
+    <value>150000</value>
```

Comment:
Is this needed?

Reply:
Never mind. See later you have added another cache.

Reply:
Yup, just wanted to make sure we have the same number of entries to preserve similar behavior.